exporter/datadogexporter/metadata/ec2/ec2.go

```diff
@@ -23,11 +24,15 @@ import (
 	"go.uber.org/zap"
 )
 
-var defaultPrefixes = [3]string{"ip-", "domu", "ec2amaz-"}
+var (
+	defaultPrefixes = [3]string{"ip-", "domu", "ec2amaz-"}
+	ec2TagPrefix    = "ec2.tag."
```

Comment:
This needs to match the prefix defined in #1899, see [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/master/processor/resourcedetectionprocessor/internal/aws/ec2/ec2.go#L35)